sdk/cosmos/azure-cosmos-encryption/pom.xml

```diff
@@ -49,7 +49,7 @@ Licensed under the MIT License.
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-security-keyvault-keys</artifactId>
-      <version>4.3.0-beta.1</version> <!-- {x-version-update;com.azure:azure-security-keyvault-keys;current} -->
+      <version>4.2.1</version> <!-- {x-version-update;com.azure:azure-security-keyvault-keys;current} -->
```

Comment:
@kushagraThapar could you confirm if this needs to be a current dependency or dependency version should work?

Reply:
This is just for the from source pipeline to work. This change won't go into master.

Reply:
as long as cosmos `Encryption_Integration_Tests_Java8` test pass we are fine